components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -307,14 +307,17 @@ PublisherInfoDatabase::GetPanelPublisher(
   }
 
   sql::Statement info_sql(db_.GetUniqueStatement(
-      "SELECT pi.publisher_id, pi.name, pi.url, pi.favIcon, pi.provider, "
-      "pi.verified, pi.excluded, ai.percent FROM publisher_info AS pi "
-      "LEFT JOIN activity_info AS ai ON pi.publisher_id = ai.publisher_id "
-      "WHERE pi.publisher_id=? AND "
-      "(ai.reconcile_stamp = ? OR ai.percent IS NULL) LIMIT 1"));
+      "SELECT pi.publisher_id, pi.name, pi.url, pi.favIcon, "
+      "pi.provider, pi.verified, pi.excluded, "
+      "("
+      "SELECT IFNULL(percent, 0) FROM activity_info WHERE "
```

Comment:
What is the subquery for? Why are we filtering on activity table (percent, reconcileStamp) info if we're just pulling panel info? Shouldn't we just be able to pull it from publisher_info?

Reply:
ignore. I know the answer to that.